calyx-frontend/src/syntax.pest

```diff
@@ -103,15 +107,15 @@ sig_with_params = {
   params? ~ signature
 }
 primitive = {
-     comb? ~ "primitive" ~ name_with_attribute ~ sig_with_params ~ ";"
+     (comb | static_annotation)? ~ "primitive" ~ name_with_attribute ~ sig_with_params ~ ";"
```

Comment:
Moving the `(comb | static_annotation)` stuff into a new rule would help with the thing I mentioned in `parser.rs`